mocked-api/schemasData.ts

```diff
@@ -0,0 +1,702 @@
+    ],
+  },
+  "aws_lambda_sink_0.1": {
+    type: "object",
+    additionalProperties: false,
+    required: [
+      "aws_function",
+      "aws_region",
+      "aws_access_key",
+      "aws_secret_key",
+    ],
+    properties: {
+      aws_function: {
+        title: "Function Name",
+        description: "The Lambda Function name",
+        type: "string",
+      },
+      aws_access_key: {
+        title: "Access Key",
+        "x-group": "credentials",
+        oneOf: [
+          {
+            title: "Access Key",
+            description: "The access key obtained from AWS",
+            type: "string",
+            format: "password",
+          },
+          {
+            description: "An opaque reference to the aws_access_key",
+            type: "object",
+            properties: {},
+          },
+        ],
+      },
+      aws_secret_key: {
+        title: "Secret Key",
+        "x-group": "credentials",
+        oneOf: [
+          {
+            title: "Secret Key",
+            description: "The secret key obtained from AWS",
+            type: "string",
+            format: "password",
+          },
+          {
+            description: "An opaque reference to the aws_secret_key",
+            type: "object",
+            properties: {},
+          },
+        ],
+      },
+      aws_region: {
+        title: "AWS Region",
+        description: "The AWS region to connect to",
+        type: "string",
+        example: "eu-west-1",
+        enum: [
+          "af-south-1",
+          "ap-east-1",
+          "ap-northeast-1",
+          "ap-northeast-2",
+          "ap-northeast-3",
+          "ap-south-1",
+          "ap-southeast-1",
+          "ap-southeast-2",
+          "ap-southeast-3",
+          "ca-central-1",
+          "eu-central-1",
+          "eu-north-1",
+          "eu-south-1",
+          "eu-west-1",
+          "eu-west-2",
+          "eu-west-3",
+          "fips-us-east-1",
+          "fips-us-east-2",
+          "fips-us-west-1",
+          "fips-us-west-2",
+          "me-south-1",
+          "sa-east-1",
+          "us-east-1",
+          "us-east-2",
+          "us-west-1",
+          "us-west-2",
+          "cn-north-1",
+          "cn-northwest-1",
+          "us-gov-east-1",
+          "us-gov-west-1",
+          "us-iso-east-1",
+          "us-iso-west-1",
+          "us-isob-east-1",
+        ],
+      },
+    },
+  },
+  "aws_s3_source_0.1": {
+    type: "object",
+    additionalProperties: false,
+    required: [
+      "aws_bucket_name_or_arn",
+      "aws_region",
+      "aws_access_key",
+      "aws_secret_key",
+    ],
+    properties: {
+      aws_bucket_name_or_arn: {
+        title: "Bucket Name",
+        description: "The S3 Bucket name or ARN",
+        type: "string",
+      },
+      aws_delete_after_read: {
+        title: "Auto-delete Objects",
+        description: "Delete objects after consuming them",
+        type: "boolean",
+        default: true,
+      },
```

Comment:
We miss help icon for boolean fields. 

![Screenshot from 2022-06-16 08-31-10](https://user-images.githubusercontent.com/8044780/174006355-e1ff9c94-21e7-4aaa-863e-33a18c7d9f18.png)

Reply:
There's not much we can do about this (console warning included).

Help icons (+ tooltips) are generated by the PF `FormGroup` component that wraps the input field. In order to display the help icon a label must be present, like **Bucket Name** in the screenshot.

Boolean fields are rendered with a PF `Checkbox`. In this case no label is passed to `FormGroup` because its value is used as label for the `Checkbox` itsefl. That's understandable, otherwise we would have the same label two times.

So, no label to the input wrap, no help icon and no tooltip.